test/extensions/compression/gzip/compressor/BUILD

```diff
@@ -16,8 +20,8 @@ envoy_cc_fuzz_test(
     deps = [
```

Comment:
I'd put the fuzz test one directory up, because it is testing both compression and decompression.